apps/meteor/app/livechat/server/api/v1/config.ts

```diff
@@ -18,17 +19,17 @@ API.v1.addRoute(
 				return API.v1.success({ config: { enabled: false } });
 			}
 
-			const { token, department, businessUnit } = this.queryParams;
+			const { token, department, businessUnit } = this.queryParams as { token?: string; department?: string; businessUnit?: string };
```

Comment:
Since this is on TS, shouldn't the type come from rest typings?

Reply:
Lol! I totally forgot about that :joy: Let me check and remove this